scriptworker.yaml.tmpl

```diff
@@ -0,0 +1,145 @@
+#-----------------------------------------------------------------------------------------------
+# Taskcluster worker settings.
+# For development, keep the test-dummy-* and dummy-worker-* convention.
+# In production, follow the production conventions.
+#-----------------------------------------------------------------------------------------------
+provisioner_id: test-dummy-provisioner
+worker_group: test-dummy-workers
+worker_type: dummy-worker-myname
+
+# worker_id will default to env['SCRIPTWORKER_WORKER_ID'] if not set, for CloudOps deployment.
+# We may be able to remove that default if we don't use CloudOps deployment for scriptworker
+# instances.
+worker_id: dummy-worker-myname1
+
+
+#-----------------------------------------------------------------------------------------------
+# Taskcluster credentials.
+# Uncomment and edit to specify the taskcluster credentials here.
+# Taskcluster credentials can also be set in secrets.json, $HOME/.scriptworker,
+# /etc/.scriptworker, or the environment variables TASKCLUSTER_ACCESS_TOKEN,
+# TASKCLUSTER_CLIENT_ID, and TASKCLUSTER_CERITIFICATE.
+#-----------------------------------------------------------------------------------------------
+# credentials:
+#    clientId: ...
+#    accessToken: ...
+#    certificate: ...
+
+#-----------------------------------------------------------------------------------------------
+# Task configs
+#-----------------------------------------------------------------------------------------------
+artifact_expiration_hours: 24
+
+# The timeouts are in seconds.
+artifact_upload_timeout: 1200
+task_max_timeout: 1200
+
+# This is the command line to execute the task.
+task_script: ["bash", "-c", "echo foo && sleep 19 && exit 1"]
+
+# debug logging?
+verbose: true
+
+# In tier 1 production, these should all be true.
+sign_chain_of_trust: false
+verify_chain_of_trust: false
+verify_cot_signature: false
+
+
+#-----------------------------------------------------------------------------------------------
+# Scriptworker paths.
+#-----------------------------------------------------------------------------------------------
+# Scriptworker logs go here; this is a long-lived directory.
+log_dir: "/tmp/log"
```

Comment:
I never intended /tmp to be a real location for anything but testing.
For puppetized instances, we're going with /builds/scriptworker/... and the like.
I'm not sure we gain that much from moving the `log_dir` for dev instances, and it forces someone testing to create /var/log/... and make sure it's writable for the current user.

I'm going to skip this suggestion for now, but if you feel strongly about it we can revisit.